<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="Viewer" name="viewer"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>896</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionOpen"/>
    <addaction name="actionExport"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionDocumentation"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuProjection">
     <property name="title">
      <string>&amp;Traces</string>
     </property>
     <addaction name="actionRight"/>
     <addaction name="actionVentral"/>
     <addaction name="actionAnterior"/>
     <addaction name="actionLeft"/>
     <addaction name="actionDorsal"/>
     <addaction name="actionPosterior"/>
    </widget>
    <widget class="QMenu" name="menuOverlay">
     <property name="title">
      <string>&amp;Overlay</string>
     </property>
     <addaction name="actionORight"/>
     <addaction name="actionOVentral"/>
     <addaction name="actionOAnterior"/>
     <addaction name="actionOLeft"/>
     <addaction name="actionODorsal"/>
     <addaction name="actionOPosterior"/>
    </widget>
    <widget class="QMenu" name="menuEOverlay">
     <property name="title">
      <string>Exclusive overlay</string>
     </property>
     <addaction name="actionEORight"/>
     <addaction name="actionEOVentral"/>
     <addaction name="actionEOAnterior"/>
     <addaction name="actionEOLeft"/>
     <addaction name="actionEODorsal"/>
     <addaction name="actionEOPosterior"/>
    </widget>
    <widget class="QMenu" name="menuProjection_2">
     <property name="title">
      <string>&amp;Projection</string>
     </property>
     <addaction name="actionPRight"/>
     <addaction name="actionPVentral"/>
     <addaction name="actionPAnterior"/>
     <addaction name="actionPLeft"/>
     <addaction name="actionPDorsal"/>
     <addaction name="actionPPosterior"/>
    </widget>
    <addaction name="menuProjection"/>
    <addaction name="menuOverlay"/>
    <addaction name="menuProjection_2"/>
    <addaction name="menuEOverlay"/>
    <addaction name="actionResetRotation"/>
    <addaction name="actionGotoXYZ"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCopy"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionMarkLocation"/>
    <addaction name="actionExportSubstack"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="modes">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Mouse action:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="select">
          <property name="text">
           <string>Select object</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="add">
          <property name="text">
           <string>Add to object</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="erase">
          <property name="text">
           <string>Erase pixels</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Show:</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="sall">
          <property name="text">
           <string>All objects</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="snamed">
          <property name="text">
           <string>Named objects</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="sanon">
          <property name="text">
           <string>Unnamed objects</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="snone">
          <property name="text">
           <string>Nothing</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Brightness and contrast:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="QSlider" name="whiteSlider">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSlider" name="gammaSlider">
          <property name="minimum">
           <number>-99</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="blackSlider">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>White</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Gamma</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Black</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="blackValue">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="whiteValue">
          <property name="text">
           <string>255</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="gammaValue">
          <property name="text">
           <string>1.0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="pnew">
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Go to:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pfind">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Obj ID:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="id">
          <property name="text">
           <string>01</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="pdel">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="name"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionDraw_new_object">
   <property name="text">
    <string>Draw new object</string>
   </property>
  </action>
  <action name="actionDelete_object">
   <property name="text">
    <string>Delete object</string>
   </property>
  </action>
  <action name="actionSelect_object">
   <property name="text">
    <string>Select object</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>&amp;Documentation</string>
   </property>
  </action>
  <action name="actionVentral">
   <property name="text">
    <string>Ventral</string>
   </property>
  </action>
  <action name="actionDorsal">
   <property name="text">
    <string>Dorsal</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionAnterior">
   <property name="text">
    <string>Anterior</string>
   </property>
  </action>
  <action name="actionPosterior">
   <property name="text">
    <string>Posterior</string>
   </property>
  </action>
  <action name="actionORight">
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionOVentral">
   <property name="text">
    <string>Ventral</string>
   </property>
  </action>
  <action name="actionOAnterior">
   <property name="text">
    <string>Anterior</string>
   </property>
  </action>
  <action name="actionOLeft">
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionODorsal">
   <property name="text">
    <string>Dorsal</string>
   </property>
  </action>
  <action name="actionOPosterior">
   <property name="text">
    <string>Posterior</string>
   </property>
  </action>
  <action name="actionEORight">
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionEOVentral">
   <property name="text">
    <string>Ventral</string>
   </property>
  </action>
  <action name="actionEOAnterior">
   <property name="text">
    <string>Anterior</string>
   </property>
  </action>
  <action name="actionEOLeft">
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionEODorsal">
   <property name="text">
    <string>Dorsal</string>
   </property>
  </action>
  <action name="actionEOPosterior">
   <property name="text">
    <string>Posterior</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>&amp;Export</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>&amp;Copy image to clipboard</string>
   </property>
  </action>
  <action name="actionPRight">
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionPVentral">
   <property name="text">
    <string>Ventral</string>
   </property>
  </action>
  <action name="actionPAnterior">
   <property name="text">
    <string>Anterior</string>
   </property>
  </action>
  <action name="actionPLeft">
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionPDorsal">
   <property name="text">
    <string>Dorsal</string>
   </property>
  </action>
  <action name="actionPPosterior">
   <property name="text">
    <string>Posterior</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>&amp;Import...</string>
   </property>
  </action>
  <action name="actionResetRotation">
   <property name="text">
    <string>&amp;Reset rotation</string>
   </property>
  </action>
  <action name="actionGotoXYZ">
   <property name="text">
    <string>&amp;Go to x,y,z</string>
   </property>
  </action>
  <action name="actionMarkLocation">
   <property name="text">
    <string>&amp;Mark location</string>
   </property>
  </action>
  <action name="actionExportSubstack">
   <property name="text">
    <string>&amp;Export substack...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Viewer</class>
   <extends>QWidget</extends>
   <header>Viewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
